packages/patternfly-3/patternfly-react/src/components/LoginPage/components/LoginCardComponents/LoginFormError.js

```diff
@@ -4,9 +4,11 @@ import Collapse from 'react-collapse';
 
 const LoginFormError = ({ children, show, className, ...props }) => (
   <div className={`login-form-error ${className}`}>
-    <Collapse {...props} isOpened={show}>
-      {children}
-    </Collapse>
+    {children ? (
+      <Collapse {...props} isOpened={show}>
+        {children}
+      </Collapse>
+    ) : null}
   </div>
```

Comment:
Is it possible to see this in the storybook?

Reply:
Yes, it works there on the `managed login page`, see in the build above :)

Reply:
Can we add the `login-form-error` class to `Collapse` and avoid the extra div?

Reply:
yes, great catch @jeff-phillips-18 :)